analysis/grammar_search_analysis.py

```diff
@@ -242,29 +242,33 @@ def _make_proxy_analysis_results(outdir: str) -> None:
 
 
 def _main() -> None:
-    env_names = [
-        "cover",
-        "blocks",
-        "painting",
-    ]
-    score_function_names = [
-        # "prediction_error",
-        # "hadd_energy_lookaheaddepth0",
-        # "hadd_energy_lookaheaddepth1",
-        # "hadd_energy_lookaheaddepth2",
-        # "exact_energy",
-        # "lmcut_count_lookaheaddepth0",
-        # "hadd_count_lookaheaddepth0",
-        # "exact_count",
-        "expected_nodes",
-    ]
-    run_planning = False
+    args = utils.parse_args(env_required=False,
+                            approach_required=False,
+                            seed_required=False)
+    assert args["excluded_predicates"] == ""  # gets ignored
+    if args["env"] is not None:
+        env_names = [args["env"]]
+    else:
+        del args["env"]
+        env_names = [
+            "cover",
+            "blocks",
+            "painting",
+        ]
+    if args["seed"] is None:
+        args["seed"] = 0
+    if "grammar_search_score_function" in args:
+        score_function_names = [args["grammar_search_score_function"]]
+    else:
+        score_function_names = ["expected_nodes"]
+    run_planning = True
```

Comment:
same, `RUN_PLANNING` can be a constant at top of file, and also can we set it False unless we have a good reason to have it True by default?